.ci/run-tests

```diff
@@ -8,13 +8,19 @@
 export TEST_SUITE="${TEST_SUITE:=oss}"
 export PYTHON_VERSION="${PYTHON_VERSION:=3.9}"
 export PYTHON_CONNECTION_CLASS="${PYTHON_CONNECTION_CLASS:=Urllib3HttpConnection}"
-export OPENSEARCH_URL_EXTENSION="${OPENSEARCH_URL_EXTENSION:=http}"
-export CLUSTER="${1:=opensearch}"
+export CLUSTER="${1:-opensearch}"
+export SECURE_INTEGRATION="${2:-false}"
+if [[ "$SECURE_INTEGRATION" == "true" ]]; then
```

Comment:
Could be simplified if you use it as a boolean expression:
```
if "$SECURE_INTEGRATION"; then
...
```

Reply:
Not an expert in bash, but looks like it's preferable to do comparison the way I've done, since it's a string which we get as an input to the `run-tests` file.